builder/profiles/fastboot.cmake

```diff
@@ -24,6 +24,8 @@ else()
   set( MFX_LIBNAME "mfxhw32-fastboot" )
 endif()
 
+set( MFX_PRODUCT_NAME "Intel(R) Media SDK for Embedded Linux" )
```

Comment:
I am not sure I lake this as a product name. @onabiull : maybe "Intel(R) Media SDK Fastboot"?

Reply:
@dvrogozh lets keep one name only.